src/FilePathRouter.php

```diff
@@ -1,28 +1,35 @@
 <?php
 
+declare(strict_types=1);
+
 namespace BlankFramework\FilePathRouter;
 
 use BlankFramework\FilePathRouter\Exception\RouteNotFoundException;
 use BlankFramework\FilePathRouter\Exception\RoutesPathNotFoundException;
 
-class FilePathRouter
+final class FilePathRouter
 {
     private string $routesPath;
 
+
+    /**
+     * @throws RoutesPathNotFoundException
+     */
     public function __construct(string $routesPath)
     {
         $this->setRoutesPath($routesPath);
     }
 
-    private function setRoutesPath(string $routesPath): void
-    {
-        if (!$this->routeExists($routesPath)) {
-            throw new RoutesPathNotFoundException($routesPath);
-        }
-
-        $this->routesPath = rtrim($routesPath, '/');
-    }
 
+    /**
+     * Routes the request to the appropriate file based on the given path.
+     *
+     * @param string $path The request path.
+     *
+     * @return string The path to the route file.
+     *
```

Comment:
Remove all but the `@return`. Change return description to The absolute path to the index.php file that you should call in your application.